src/bin/find.js

```diff
@@ -42,7 +45,7 @@ const finderOptions = {
   omitCore: !argv.core,
   includeDeprecated: argv.include === 'deprecated',
   ext: argv.ext,
-  useFlatConfig: argv.flatConfig
+  useFlatConfig: semver.satisfies(eslintPkg.version, '>= 8.57') && argv.flatConfig
```

Comment:
all this PR seems to do is prevent `useFlatConfig` from being `true` in an eslint without flat config support